src/app/Fake.DotNet.Paket/Paket.fs

```diff
@@ -32,9 +32,24 @@ type PaketPackParams =
       MinimumFromLockFile : bool
       PinProjectReferences : bool }
 
+let private findPaketExecutable () =
+    let windowsName = "paket.exe"
+    let unixName = "paket"
+
+    let exeName =
+        if Environment.isWindows then
+            windowsName
+        else
+            if !! ("./**/" @@ unixName) |> Seq.isEmpty |> not then
```

Comment:
if we already search for it why not use that result but search again?

Reply:
We might need to introduce a `tryFindToolFolderInSubPath` and call that first and retry with the `.exe`

Reply:
I thought about that. I can try that out instead, but what should the fallback be if it can't find any of them?

Reply:
I'd problably use `try*` for the first and the other overload for the second call (ie. just use the default `paket.exe` as we do now)